File: engine/model/src/main/scala/net/server/channel/packet/ring/RespondToProposalPacket.scala
package net.server.channel.packet.ring

class RespondToProposalPacket(private var _mode: Byte, private var _accepted: Boolean, private var _name: String, private var _itemId: Int) extends BaseRingPacket(_mode = _mode) {
  def accepted: Boolean = _accepted

  def name: String = _name

  def itemId: Int = _itemId
}

File: engine/engine-model/src/main/java/net/server/channel/packet/ring/RespondToProposalPacket.java
package net.server.channel.packet.ring;

public class RespondToProposalPacket extends BaseRingPacket {
   private final Boolean accepted;

   private final String name;

   private final Integer itemId;

   public RespondToProposalPacket(Byte mode, Boolean accepted, String name, Integer itemId) {
      super(mode);
      this.accepted = accepted;
      this.name = name;
      this.itemId = itemId;
   }

   public Boolean accepted() {
      return accepted;
   }

   public String name() {
      return name;
   }

   public Integer itemId() {
      return itemId;
   }
}
